### PR TITLE
Localize “email address” placeholder

### PR DESCRIPTION
**Pull request by Arhell**
issue #20548
/language vi

### CODE REVIEW

File: i18n/vi.toml [hunk 1]
```diff
@@ -57,6 +57,9 @@ other = "Trang này có hữu ích?"
 [feedback_yes]
 other = "Có"
 
+[input_placeholder_email_address]
+other = "địa chỉ email"
```
Comment (truongnh1992) [inline i18n/vi.toml:61]: It should be: "địa chỉ email"

Comment (Arhell) [reply i18n/vi.toml]: done
Thank!

**Comment by truongnh1992** [comment]
/lgtm
/approve
Thank you.

**Comment by sftim** [comment]
Echoing https://github.com/kubernetes/website/pull/20562#issuecomment-620319543
/approve

**Pull request merged**